client/public/src/components/Words.jsx

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
 import Word from './Word.jsx';
 
-const Words = ({ text, keyWords, className}) => {
+const Words = (props) => {
+  const {text, keyWords, className} = props;
   return (
     <div className={className}>
       {
-        text.split(' ').map((word, index) => {
-          return (
-            <Word 
-              word={word}
-              keyWords={keyWords}
-              key={index}
-              index={index}/>
-          )
-        })
+        keyWords.length ? (
```

Comment:
This is just a style thing, but for stuff like this, unless it's really clear and one line, i've been told to stay away from ternary.